pkg/kubenest/manifest/controlplane/apiserver/mainfests_deployment.go

```diff
@@ -128,5 +128,356 @@ spec:
+          name: etcd-cert
+          readOnly: true
+        - mountPath: /etc/kubernetes/konnectivity-server
+          readOnly: false
+          name: konnectivity-uds
+        - name: kas-proxy
+          mountPath: /etc/kubernetes/konnectivity-server-config/{{ .Namespace }}/egress_selector_configuration.yaml
+          subPath: egress_selector_configuration.yaml
+      - name: konnectivity-server-container
+        image: {{ .ImageRepository }}/kas-network-proxy-server:{{ .Version }}
+        resources:
+          requests:
+            cpu: 1m
+        securityContext:
+          allowPrivilegeEscalation: false
+          runAsUser: 0
+        command: [ "/proxy-server"]
+        args: [
+          "--log-file=/var/log/konnectivity-server.log",
+          "--logtostderr=true",
+          "--log-file-max-size=0",
+          "--uds-name=/etc/kubernetes/konnectivity-server/{{ .Namespace }}/konnectivity-server.socket",
+          "--delete-existing-uds-file",
+          "--cluster-cert=/etc/virtualcluster/pki/apiserver.crt",
+          "--cluster-key=/etc/virtualcluster/pki/apiserver.key",
+          "--server-port=0",
+          "--agent-port={{ .AgentPort }}",
+          "--health-port=8092",
+          "--admin-port=8093",
+          "--keepalive-time=1h",
+          "--mode=grpc",
+          "--agent-namespace=kube-system",
+          "--agent-service-account=konnectivity-agent",
+          "--kubeconfig=/etc/apiserver/kubeconfig",
+          "--authentication-audience=system:konnectivity-server",
+          ]
+        livenessProbe:
+          httpGet:
+            scheme: HTTP
+            host: 127.0.0.1
+            port: 8092
+            path: /healthz
+          initialDelaySeconds: 10
+          timeoutSeconds: 60
+        ports:
+        - name: serverport
+          containerPort: 8090
+        - name: agentport
+          containerPort: {{ .AgentPort }}
+          hostPort: {{ .AgentPort }}
+        - name: healthport
+          containerPort: 8092
+        - name: adminport
+          containerPort: 8093
+        volumeMounts:
+        - mountPath: /etc/virtualcluster/pki
+          name: apiserver-cert
+          readOnly: true
+        - name: varlogkonnectivityserver
+          mountPath: /var/log/konnectivity-server.log
+          readOnly: false
+        - name: kubernetes
+          mountPath: /etc/kubernetes
+          readOnly: true
+        - name: konnectivity-home
+          mountPath: /etc/kubernetes/konnectivity-server
+        - mountPath: /etc/apiserver/kubeconfig
+          name: kubeconfig
+          subPath: kubeconfig
+      priorityClassName: system-node-critical
+      volumes:
+      - name: kubeconfig
+        secret:
+          defaultMode: 420
+          secretName: {{ .KubeconfigSecret }}
+      - name: varlogkonnectivityserver
+        hostPath:
+          path: /var/log/konnectivity-server.log
+          type: FileOrCreate
+      - name: kubernetes
+        hostPath:
+          path: /etc/kubernetes
```

Comment:
Is this necessary？

Reply:
it used by `        - --egress-selector-config-file=/etc/kubernetes/konnectivity-server-config/{{ .Namespace }}/egress_selector_configuration.yaml
`

Reply:
removed